kouncil-frontend/apps/kouncil/src/app/schemas/list/schemas.component.ts

```diff
@@ -0,0 +1,231 @@
+import {TopicsService} from '@app/feat-topics';
+import {Topics} from '@app/common-model';
+import {Router} from '@angular/router';
+
+@Component({
+  selector: 'app-schemas',
+  template: `
+    <div class="main-container">
+      <div class="toolbar-container">
+        <div class="toolbar">
+          <app-common-autocomplete [control]="topicFilterControl"
+                                   [data]="topicList"
+                                   [placeholder]="'Topics'"
+                                   [emptyFilteredMsg]="'No topics found'"
+                                   [panelWidth]="'auto'"
+                                   (selectedValueEvent)="updateTopics($event)"></app-common-autocomplete>
+
+          <button mat-button class="action-button-black" (click)="search()">
+            Search
+            <mat-icon class="search-icon">search</mat-icon>
+          </button>
+
+          <button mat-button class="action-button-white" (click)="clearFilters()">
+            Clear filters
+          </button>
+
+          <button mat-button *ngIf="authService.canAccess([KouncilRole.KOUNCIL_EDITOR])"
+                  class="action-button-black" [routerLink]="['/schemas/create']">
+            Add new schema
+          </button>
+        </div>
+      </div>
+    </div>
+
+    <div *ngIf="filtered">
+      <ng-template #noDataPlaceholder>
+        <app-no-data-placeholder [objectTypeName]="'Schemas'"></app-no-data-placeholder>
+      </ng-template>
+
+      <app-common-table *ngIf="filtered && filtered.length > 0; else noDataPlaceholder"
+                        [tableData]="filtered" [columns]="columns" matSort [sort]="sort"
+                        cdkDropList cdkDropListOrientation="horizontal"
+                        (cdkDropListDropped)="drop($event)" [actionColumns]="actionColumns"
+                        (rowClickedAction)="navigateToDetails($event)">
+
+        <ng-container *ngFor="let column of columns; let index = index">
+          <app-common-table-column [column]="column" [index]="index"></app-common-table-column>
+        </ng-container>
+
+        <ng-container *ngFor="let column of actionColumns; let index = index">
+          <app-common-table-column [column]="column"
+                                   [index]="index + columns.length"
+                                   [template]="cellTemplate">
+
+            <ng-template #cellTemplate let-element>
+              <div class="actions-column">
+                <button class="action-button"
+                        *ngIf="authService.canAccess([KouncilRole.KOUNCIL_EDITOR])"
+                        [routerLink]="['/schemas/edit/' + element.subjectName + '/' + element.version]">
+                  Edit
+                </button>
+                <button class="action-button"
+                        *ngIf="authService.canAccess([KouncilRole.KOUNCIL_EDITOR])"
+                        (click)="deleteSchema(element.subjectName, element.version)">
```

Comment:
I created schema X (version 1), then edited (version 2) and when I deleted schema and created again it had version 3, is it expected behaviour?